### PR TITLE
Mat 5554 handle qdm scoring change

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-5554](https://jira.cms.gov/browse/MAT-5554)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx [hunk 1]
```diff
@@ -87,12 +100,20 @@ const BaseConfiguration = () => {
...
+      measure.groups = null;
+    }
     const newMeasure: Measure = {
       ...measure,
-      scoring: values.scoring,
-      baseConfigurationTypes: values.baseConfigurationTypes,
-      patientBasis: values.patientBasis,
+      scoring: formik.values.scoring,
+      baseConfigurationTypes: formik.values.baseConfigurationTypes,
+      patientBasis: formik.values.patientBasis === "true" ? true : false,
```
Comment (RohitKandimalla) [inline src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx]:
```suggestion
      patientBasis: formik.values.patientBasis === "true",
```

Comment (RohitKandimalla) [reply src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx]: I believe we can just do a comparison and the comparison operator is going to return us back the boolean values.

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx [hunk 2]
```diff
@@ -39,6 +43,11 @@ const BaseConfiguration = () => {
   const [toastMessage, setToastMessage] = useState<string>("");
   const [toastType, setToastType] = useState<string>("danger");
   const { updateRouteHandlerState } = routeHandlerStore;
+  const [currentScoring, setCurrentScoring] = useState<string>();
+  const [changeScoringDialog, setChangeScoringDialog] =
+    useState<ChangeScoringDialog>({
+      open: false,
```
Comment (RohitKandimalla) [inline src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx]: Any reason we need a ChangeScoringDialog instead of directly using a boolean value ?

**Review by RohitKandimalla — CHANGES_REQUESTED** [review]

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx (hunk 2 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx]: @RohitKandimalla Just in case we need any other variations, e.g. messages based on scoring, etc.

Comment (RohitKandimalla) [reply src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx]: Yeah but if we don't need, then this will an overkill. But yeah sure. it's not a show-stopper.

**Review by RohitKandimalla — DISMISSED** [review]

**Review by mcmcphillips — DISMISSED** [review]

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx (hunk 2 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx]: @RohitKandimalla Just changed it to boolean. Thanks Rohit!

**Review by RohitKandimalla — APPROVED** [review]
